images/dockerfiles/stellar-core-build/build.sh

```diff
@@ -11,8 +11,7 @@ cd /stellar-core
 # make format
 
 # build
-make -j $(nproc)
-
+make -j $(nproc) check && make -j $(nproc)
 # test
 # export ALL_VERSIONS=1
 # make check
```

Comment:
- check if you can use `ALL_VERSIONS` and if they're using anything else new since the last time i looked at it.
- remove obsolete comments

Reply:
see the following files upstream:
- https://github.com/stellar/stellar-core/blob/master/travis-build.sh
- https://github.com/stellar/stellar-core/blob/master/.travis.yml
- https://github.com/stellar/stellar-core/blob/master/INSTALL.md